app/services/feeds/hacker_news/serviceable.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Feeds
+  module HackerNews
+    # The base URI for our service provider's feed API.
+    module Serviceable
+      # TODO: Pull this from a configurable source.
+      BASE_URI = 'https://hacker-news.firebaseio.com'
+    end
+  end
+end
```

Comment:
A simple module that provides the base URI for a particular news service API. In the real world, we would want this to be configurable.